pkg/fileutil/fileutil.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fileutil
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/pkg/device"
+)
+
+func copyRegularFile(src, dest string) (err error) {
+	srcFile, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer srcFile.Close()
```

Comment:
I don't think we need to handle an error on closing a [read-only file](https://golang.org/pkg/os/#Open). However, handling the error when closing `destFile` makes sense because it can cause silent loss of data.